docs/getting-started.md

```diff
@@ -259,6 +261,13 @@ configuration file has the following fields:
 ```
 </details>
 
+The default `shim_base_dir` is /var/lib/firecracker-containerd/shim-base and is
+required to be created before running firecracker-containerd.
+
+```bash
+mkdir -p /var/lib/firecracker-containerd/shim-base
```

Comment:
We only require the user to create `/var/lib/firecracker-containerd`, right?

Reply:
Fixed!